codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/ProtoToGraphNodeMappings.scala

```diff
@@ -3,36 +3,21 @@ package io.shiftleft.codepropertygraph.cpgloading
 import flatgraph.*
 import io.shiftleft.proto.cpg.Cpg.CpgStruct
 
+import scala.collection.mutable
 import scala.jdk.CollectionConverters.*
 
-/** Mutable datastructure to preserve mapping between proto and cpg nodes during ProtoToCpg import.
+/** Mutable data structure to preserve mapping between proto and cpg nodes during ProtoToCpg import.
   *
   * Context: we need to run two passes: 1) add nodes and 2) set node properties and add edges (this is due to
   * flatgraph-specific implementation details)
   *
-  * Because of that, we need to remember the mapping from proto node id to gnode. Typically that's just a plain mapping,
-  * but there's one special case for TYPE nodes: some (parallel) frontends create duplicate TYPE nodes which we need to
-  * deduplicate...
+  * Because of that, we need to remember the mapping from proto node id to gnode. Typically, that's just a plain
+  * mapping. But there's one special case for TYPE nodes: some (parallel) frontends create duplicate TYPE nodes which we
+  * need to deduplicate...
   */
 class ProtoToGraphNodeMappings {
-  private var protoNodeIdToGNode  = Map.empty[Long, DNode]
-  private var typeFullNameToGNode = Map.empty[String, DNode]
-
-  def addAll(other: ProtoToGraphNodeMappings): Unit = {
```

Comment:
removing this since it appears unused, not because there's any problem implementing it with the mutable maps